src/uct/ib/dc/accel/dc_mlx5.c

```diff
@@ -801,6 +801,9 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h md, uct_worker_h worker
                              UCT_IB_MLX5_AV_FULL_SIZE) /
                              sizeof(struct mlx5_wqe_data_seg));
 
+    uct_rc_mlx5_iface_common_prepost_recvs(&self->super.super,
```

Comment:
yes, dc_mlx5 does not support having smaller initial receive queue length in this PR. It requires special handling because of the SRQ usage in mlx5, so would be in a separate PR.